CHANGES.md

```diff
@@ -15,6 +15,11 @@
   directory path in the scope of the `DatastoreManager` class.
 - [FIX] Corrected a case where two root nodes with identical revision IDs prevented selection of the
   correct new winning revision.
+- [FIX] Added migration on `Datastore` opening to repair datastores that have documents with
+  multiple identical revision IDs (as caused by issue 329). The migration will delete duplicate
```

Comment:
Since we have tended not to mention issue numbers or go into huge internal detail how about simply:
>- [FIX] Added migration on `Datastore` opening to repair datastores with duplicated revisions or attachments caused by an issue running parallel pull replications with the same source and target.